test/k8s/egress.go

```diff
@@ -88,8 +88,6 @@ var _ = SkipDescribeIf(func() bool {
 		// different configurations for the tests.
 		ciliumFilename = helpers.TimestampFilename("cilium.yaml")
 		DeployCiliumOptionsAndDNS(kubectl, ciliumFilename,
```

Comment:
```suggestion
		DeployCiliumAndDNS(kubectl, ciliumFilename)
```

(i.e. we should stop setting `enableCiliumEndpointSlice=false` in the configmap)

Reply:
Fixed. Thx~